frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -48,13 +48,19 @@ public void startWorker(int port) throws WorkerInitializationException, Interrup
             throw new WorkerInitializationException("Failed get TS home directory", e);
         }
 
-        String[] args = new String[6];
+        String[] args = new String[12];
         args[0] = EnvironmentUtils.getPythonRunTime(model);
-        args[1] = new File(workingDir, "ts/model_service_worker.py").getAbsolutePath();
-        args[2] = "--sock-type";
-        args[3] = connector.getSocketType();
-        args[4] = connector.isUds() ? "--sock-name" : "--port";
-        args[5] = connector.getSocketPath();
+        args[1] = "-m";
+        args[2] = "intel_extension_for_pytorch.cpu.launch";
+        args[3] = "--ninstances";
+        args[4] = "1";
+        args[5] = "--ncore_per_instance";
+        args[6] = "22";
```

Comment:
It's better to config these parameters in config.properties.

Reply:
Hi @lxning , Could you help us where to add the code to read new config knobs from the config.properties file?
Yes, both of them are model-level configuration. They only take effects on the task launched by `ts/model_service_worker.py` script.

Reply:
@HamidShojanazeri this launch script is actually a wrapper of numactl command. handler script is supposed to be launched by it to make CPU binding take effect.